packages/migrations/src/migration.ts

```diff
@@ -0,0 +1,132 @@
+import { Provider, TxData } from '@0xproject/types';
```

Comment:
This file shouldn't exist. Merge issue. The V2 migration file is now at `src/v2/migration.ts`